### PR TITLE
Update mysql database clean up script by adding missing tables that needs to be truncated

### PR DESCRIPTION
**Pull request by DinikaSen**
## Purpose
This PR adds the following tables to the MYSQL database clean up script as they need to be truncated.
- IDN_AUTH_SESSION_META_DATA
- IDN_AUTH_SESSION_APP_INFO
- IDN_AUTH_USER_SESSION_MAPPING

### CODE REVIEW

File: common/is/clean-database.sql [hunk 1]
```diff
@@ -6,5 +6,8 @@ TRUNCATE TABLE IDN_AUTH_TEMP_SESSION_STORE;
 TRUNCATE TABLE IDN_OAUTH2_AUTHORIZATION_CODE;
 TRUNCATE TABLE IDN_OAUTH2_ACCESS_TOKEN_SCOPE;
 TRUNCATE TABLE IDN_OAUTH2_ACCESS_TOKEN;
+TRUNCATE TABLE IDN_AUTH_SESSION_META_DATA;
+TRUNCATE TABLE IDN_AUTH_SESSION_APP_INFO;
+TRUNCATE TABLE IDN_AUTH_USER_SESSION_MAPPING;
```
Comment (Pushpalanka) [inline common/is/clean-database.sql:11]: Shall we add these to the MSSQL script as well? As I saw these are not present there.

Comment (DinikaSen) [reply common/is/clean-database.sql:11]: As far as I know @ashendes is working on that. @ashendes please confirm.

Comment (ashendes) [reply common/is/clean-database.sql:11]: Will add these.

**Review by Pushpalanka — APPROVED** [review]

File: common/is/clean-database.sql (hunk 1 above)
Comment (ashendes) [reply common/is/clean-database.sql:11]: Added with 6494e38e9f2e174834780416a23942b407a19739

Comment (ashendes) [reply common/is/clean-database.sql:11]: Added with https://github.com/wso2/performance-is/pull/107

**Pull request merged**